workshop_materials.md

```diff
@@ -0,0 +1,104 @@
+# Workshop Materials
+
+This page is meant to act as a collection of the latest Imaging Applications team workshop materials.
+
+## OMERO-related workshops
+
+### Introduction to OMERO
+
+[OMERO](https://www.openmicroscopy.org/omero/) is an image data management and hosting solution. This workshop covers how to store, view, share, annotate, and analyze image data in OMERO.  
+No prior experience with OMERO is necessary.
+
+[Workshop materials](https://thejacksonlaboratory.github.io/omero-training-materials/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/omero-training-materials)
+
+### [OMERO for Python Users](#omero-for-python-users-1)
```

Comment:
I thought this fit better under Python so I linked here, but can swap it around the other way.

Reply:
I like the link here to the actual workshop materials in the Python section, keeps it clean.